test/integration/maincluster_test.go

```diff
@@ -1167,32 +1164,13 @@ experimental:
       - arn:aws:elasticloadbalancing:eu-west-1:xxxxxxxxxxxx:targetgroup/manuallymanagedetg/xxxxxxxxxxxxxxxx
     securityGroupIds:
       - sg-12345678
-  dex:
+  oidc:
     enabled: true
-    url: "https://dex.example.com"
-    clientId: "example-app"
-    username: "email"
+    url: "https://accounts.google.com"
+    clientId: "kubernetes"
+    username: "sub"
```

Comment:
Sure, no problem, didn't use Google yet. Got the config from the kubernetes's documentation. This will work for dex also.